### DISCUSSION TITLE
Menu icon different icons

### DISCUSSION THREAD

**Discussion by Jack77777777** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#669] [Q&A] [answered]
I would like to have different icons for my menu but there is always the same. Can someone point me to my mistake.

Picture where the broom icon is shown for all instead of the defined under `icon_mapping:`
![image](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/64719909/e1192a92-a0cd-4fee-8c76-bb0414d1efa6)

**Comment by PiotrMachowski** [comment, marked as answer]
You used translated values instead of raw ones. Use dev tools to check raw values:
[![Open your Home Assistant instance and show your state developer tools.](https://my.home-assistant.io/badges/developer_states.svg)](https://my.home-assistant.io/redirect/developer_states/)

**Reply by Jack77777777** [reply]
That fixed it. Thank you! Also for the amazing card!
![image](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/64719909/dd29862b-a869-4f9e-897d-efe8204a6cea)
